docs/source/aryn_cloud/aryn_partitioning_service.md

```diff
@@ -102,6 +107,8 @@ The available options are listed below:
 * ```use_ocr```: If ```true```, the partitioner uses OCR to extract text from the PDF. It defaults to ```false```, where the partitioner attempts to directly extract the text from the underlying PDF in the bounding box. It currently uses Tesseract for extraction.
 * `extract_table_structure`: If `true`, the partitioner runs a separate table extraction model to extract cells from regions of the document identified as tables.
 * `extract_images`: If `true`, the partitioner crops each region identified as an image and attaches it to the associated `ImageElement`. This can later be fed into the `SummarizeImages` transform when used within Sycamore.
+* `selected_pages`: You can specify a page (like [11]) or a page range (like [25,30]) of your PDF to process.
```

Comment:
I'd add here that the first page is 1, not 0

Reply:
Added.